tox.ini

```diff
@@ -8,7 +8,7 @@ deps = -rtest_requirements.txt
 
 [testenv:flake8]
 deps =
-    flake8==3.5.0
+    flake8
```

Comment:
The Flake8 test failed... Looks like the newer version of flake8 is finding some problems now?

```
flake8 runtests: commands[0] | flake8 cookiecutter tests setup.py
tests/test_generate_context.py:90:32: W605 invalid escape sequence '\('
tests/test_generate_context.py:90:41: W605 invalid escape sequence '\)'
```

Not sure I understand the issue though... Should we change this line [to `r'...'` string](https://lintlyci.github.io/Flake8Rules/rules/W605.html)?

https://github.com/audreyr/cookiecutter/blob/3bc7b987e4ae9dcee996ae0b00375c1325b8d866/tests/test_generate_context.py#L90